src/adapters/osm_schedule.js

```diff
@@ -31,10 +31,27 @@ function nextTransitionTime(seconds, nextTransitionDate) {
   return false
 }
 
+function hourToDate(hour) {
+  return new Date(new Date().toDateString() + ' ' + hour)
```

Comment:
i dont trust date dateString parsing, can we use arithmetic instead ?
`return new Date(Date.now() + hour * 60)`

Reply:
hour is a String : Date.now() + "06:30" * 60 > NaN